### ISSUE TITLE
Inspection result problems shouldn't print the help URL twice

### ISSUE THREAD

**Issue by hhyyrylainen** [Revolutionary-Games/Thrive#5173]
Example in the CI output:
```
Problem with no physical location detected: File is required to end with a single newline character
Microsoft.CodeAnalysis.Sarif.Result
Code should not contain blank lines at the end of the file.
Scripts/WikiUpdater.cs:697 Auto-property accessor 'NoticeSceneName.get' is never used type: UnusedAutoPropertyAccessor.Local
Offending code (offset 27755) for previous message: 'get;'
Problem with no physical location detected: Auto-property accessor 'NoticeSceneName.get' is never used
Microsoft.CodeAnalysis.Sarif.Result
Accessor in auto-property is never used
src/thriveopedia/pages/ThriveopediaWikiPage.cs:125 Method 'GeneratePage' return value is never used type: UnusedMethodReturnValue.Local
Offending code (offset 4150) for previous message: 'T'
Problem with no physical location detected: Method 'GeneratePage' return value is never used
Microsoft.CodeAnalysis.Sarif.Result
Method return value is never used
Scripts/WikiUpdater.cs:290 Indent size is incorrect, expected indent 4 spaces type: WrongIndentSize
Offending code (offset 12432) for previous message: '                '
Problem with no physical location detected: Indent size is incorrect, expected indent 4 spaces
Microsoft.CodeAnalysis.Sarif.Result
Rule has no full description
https://www.jetbrains.com/resharperplatform/help?Keyword=WrongIndentSize
https://www.jetbrains.com/resharperplatform/help?Keyword=WrongIndentSize
Code inspection detected issues, see inspect_results.xml
```

Can probably fix by storing the help text for a few lines and suppressing further help output about the same problem if the help text is the same as already printed.

**Comment by hhyyrylainen** [comment]
The latest submodule is not yet merged into master but this issue is fully completed once that is done.
